config/sample.config.yaml

```diff
@@ -33,17 +33,24 @@ app:
   disable_orgs_listing: false
   # disable_orgs_listing if set to true will disallow non-admin APIs to list all users
   disable_users_listing: false
-  # invitation_with_roles if set to true will allow people in org with the permission to send invitation to users
-  # with set of role ids. When the invitation is accepted, the user will be added to the org with the roles specified
-  # This can be a security risk if the user who is inviting is not careful about the roles he is adding
-  # and cause permission escalation
-  # Note: this is dangerous and should be used with caution
-  invitation_with_roles: false
+  
+  # configs for user invitation to join an organization
+  invite:
+    # invitation_with_roles if set to true will allow people in org with the permission to send invitation to users
+    # with set of role ids. When the invitation is accepted, the user will be added to the org with the roles specified
+    # This can be a security risk if the user who is inviting is not careful about the roles he is adding
+    # and cause permission escalation
+    # Note: this is dangerous and should be used with caution
+    invitation_with_roles: false
```

Comment:
we can just call it `with_roles` if we are putting it here? Also to make it backward compatible, lets add a small hook in `config.go` that checks if the parent `invitation_with_roles` is also set.